pkg/urlutil/git.go

```diff
@@ -8,11 +11,13 @@ var (
 		"github.com/",
 		"git@",
 	}
+
+	urlPathWithFragmentSuffix = regexp.MustCompile(".git(:?#.+)?$")
```

Comment:
`:?` at the beginning of a regular expression means "don't save this in the group", because we don't really care about what's in the fragment in there.